frontend/src/components/maps/leaflet/Spiderfier.ts

```diff
@@ -71,6 +71,7 @@ export class Spiderfier {
 
     // only one cluster expanded at a time
     if (this.cluster === cluster || cluster == null) {
+      this.cluster = null;
```

Comment:
also when the spiderfied results are closed currently it can't be re-opened, this fixes that.